internal/pkg/state/registry/registry.go

```diff
@@ -206,6 +225,12 @@ func (s *Registry) Set(objectState ObjectState) error {
 		return fmt.Errorf(`object "%s" already exists`, key.Desc())
 	}
 
+	if objectState.GetObjectPath() != "" {
+		if err := s.namingRegistry.Attach(key, objectState.GetPathInProject()); err != nil {
+			return err
+		}
+	}
```

Comment:
Implementacia. 
Docasne je tu `objectState.GetObjectPath() != ""`, kedze v niektorych pripadoch sa path nastavuje az dodatocne.
To sa fixne v niektorom z dalsich PR. Tu by toho bolo uz prilis vela.